src/components/StudyList/StudyTable.js

```diff
@@ -103,13 +117,37 @@ const StudyTable = ({
       key={col.key}
       content={col.name}
       textAlign={i > 0 ? 'center' : 'left'}
-      sorted={
-        columns.sorting.column === col.key ? columns.sorting.direction : null
-      }
-      onClick={handleSort(col.key)}
+      sorted={sorting.column === col.key ? sorting.direction : null}
+      onClick={handleSort(col.key, tableType)}
     />
   ));
 
+  let pageCount = Math.ceil(studies.length / STUS_PER_PAGE);
+  let paginatedList = studies.slice(
+    STUS_PER_PAGE * (page - 1),
+    STUS_PER_PAGE * (page - 1) + STUS_PER_PAGE,
+  );
+
+  const pagination = logEvent => (
+    <Table.Row textAlign="right">
+      <Table.HeaderCell colSpan={visibleCols.length + 1}>
+        <Pagination
+          size="mini"
+          firstItem={null}
+          lastItem={null}
+          prevItem={{content: <Icon name="angle left" />, icon: true}}
+          nextItem={{content: <Icon name="angle right" />, icon: true}}
+          activePage={page}
+          totalPages={pageCount}
+          onPageChange={(e, {activePage}) => {
+            setPage(activePage);
+            logEvent('pagination click');
```

Comment:
Instead of using a new event type here, I would say use the 'click' event type and wrap the pagination component in an `<Amplitude />` component with `pagination` scope.